Remotion/Core/Core/Core.Core.csproj

```
@@ -29,4 +28,7 @@
   <ItemGroup>
     <None Include="..\Development.Analyzers\bin\$(Configuration)\netstandard2.0\Remotion.Development.Analyzers.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="THIRD-PARTY-NOTICES.txt" Pack="true" PackagePath="" Visible="false" />
```

```suggestion
    <None Include="THIRD-PARTY-NOTICES.txt" Pack="true" PackagePath="" />
```
Visible, it's a regular file, no need to hide it.